modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/components/QuestionFilter.es.js

```diff
@@ -132,10 +133,22 @@ const getFilterValues = (form, tags) => {
 		} (keywords/any(x:${filterKeyword}))`;
 	}
 
+	query.creatorId = form.creatorId;
+
+	if (query.creatorId) {
+		const creatorIdFilter = `creatorId eq ${query.creatorId}`;
+
+		query.filterBy =
+			query.filterBy === void 0
```

Comment:
I never seem this `void 0` applied at portal before, very interesting but I would use something like:

At least is the common for our reality

```suggestion
			query.filterBy
```

The rest looks very good!